index.ts

```diff
@@ -83,6 +83,7 @@ async function init() {
   // --playwright
   // --eslint
   // --eslint-with-prettier (only support prettier through eslint for simplicity)
+  // --vue-devtools
```

Comment:
nit: maybe, as for the router, we can offer `--devtools` and `--vue-devtools`?

Reply:
Yeah, let's be consistent. Having both `--devtools` and `--vue-devtools` sounds good.